rust/operator-binary/src/crd/user_info_fetcher.rs

```diff
@@ -110,6 +117,34 @@ pub mod versioned {
         pub additional_group_attribute_filters: BTreeMap<String, String>,
     }
 
+    #[derive(Clone, Debug, Deserialize, Eq, JsonSchema, PartialEq, Serialize)]
+    #[serde(rename_all = "camelCase")]
+    pub struct EntraBackend {
+        /// Hostname of the token provider, defaults to `login.microsoft.com`.
+        #[serde(default = "entra_default_host_token")]
+        pub hostname_token: HostName,
+
+        /// Hostname of the user info provider, defaults to `graph.microsoft.com`.
+        #[serde(default = "entra_default_host_graph")]
+        pub hostname_graph: HostName,
+
+        /// Port of the identity provider. Defaults to 443.
+        #[serde(default = "entra_default_port")]
+        pub port: u16,
+
+        /// The Microsoft Entra tenant ID.
+        pub tenant_id: String,
+
+        /// Use a TLS connection. Should usually be set to WebPki.
+        #[serde(default = "default_tls_web_pki")]
```

Comment:
Can you please explain as a comment here why we can not serde(flatten) TlcClientDetails?